tests/pages/firefox/releasenotes.py

```diff
@@ -32,6 +33,7 @@ class FirefoxReleaseNotesPage(BasePage):
     _secondary_play_store_button_locator = (By.ID, 'download-android-secondary')
     _primary_app_store_button_locator = (By.ID, 'download-ios-primary')
     _secondary_app_store_button_locator = (By.ID, 'download-ios-secondary')
+    _sticky_promo_modal_content_locator = (By.CSS_SELECTOR, '.mzp-c-sticky-promo.mzp-a-slide-in')
```

Comment:
Why do the release notes test need the content_locator defined in the file? I don't see it defined for the other pages.

Reply:
Ooooh, I see the test failure. Poke me on Slack if you need to know how to run isolated tests locally.

Reply:
I think this was copy paste from the [original test](https://github.com/mozilla/bedrock/blob/master/tests/pages/firefox/new/sticky_promo.py#L15).